daos/sqlite/Database.php

```diff
@@ -234,6 +234,12 @@ public function __construct() {
                         'INSERT INTO version (version) VALUES (11)'
                     ]);
                 }
+                if (strnatcmp($version, '12') < 0) {
+                    \F3::get('db')->exec([
+                        'CREATE UNIQUE INDEX uid ON new_items(uid, source);',
+                        'INSERT INTO version (version) VALUES (12)'
```

Comment:
Could you use version 13 so that it is more clear that the database is current.

Reply:
fix in 7040883bfbd248dae6c5d266912379c31cfcacd5